scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/BigDLRemoteAttestationService.scala

```diff
@@ -33,84 +34,227 @@ import akka.util.Timeout
+        }
+        val appID = msg.get("app_id").mkString
+        val mrEnclave = msg.get("mr_enclave").mkString
+        val mrSigner = msg.get("mr_signer").mkString
+        var policyID = UUID.randomUUID.toString
+        val curContent = Map[String, Any] (
+          "app_id" -> appID,
+          "mr_enclave" -> mrEnclave,
+          "mr_signer" -> mrSigner
+        )
+        policyMap += (policyID -> curContent)
+        val res = AttestationUtil.mapToString(Map("policy_id" -> policyID))
+        complete(200, res)
+      } else {
+        // TODO: TDX policy
+        complete(400, "Not Implemented.")
+      }
+    }
+  }
+
+  def verifyQuote(msg: Map[String, Any]): Route = {
+    if (!checkAppIDAndApiKey(msg)) {
+      complete(400, "Invalid app_id and api_key.")
+    } else {
+      if (!msg.contains("quote")) {
+        complete(400, "Required parameters are not provided.")
+      } else {
+        val quoteBase64 = msg.get("quote").mkString
+        val quote = Base64.getDecoder().decode(quoteBase64.getBytes)
+        val verifyQuoteResult = quoteVerifier.verifyQuote(quote)
+        if (verifyQuoteResult < 0) {
+          val res = "{\"result\":\"" + verifyQuoteResult.toString() + "\"}"
+          complete(400, res)
+        } else {
+          if (!msg.contains("policy_id")) {
+            val res = "{\"result\":" + verifyQuoteResult.toString() + "}"
+            complete(200, res)
+          } else {
+            val appID = msg.get("app_id").mkString
+            val mrEnclave = AttestationUtil.getMREnclaveFromQuote(quote)
+            val mrSigner = AttestationUtil.getMRSignerFromQuote(quote)
+
+            val policyID = msg.get("policy_id").mkString
+            val policyContent: Map[String, Any] = policyMap.get(policyID) match {
+              case Some(map: Map[String, Any]) => map
+              case None => Map.empty
+            }
+            if (appID == policyContent.get("app_id").mkString &&
+              mrEnclave == policyContent.get("mr_enclave").mkString &&
+              mrSigner == policyContent.get("mr_signer").mkString) {
+              val res = "{\"result\":\"" + verifyQuoteResult.toString() + "\"}"
+              complete(200, res)
+            } else {
+              val res = "{\"result\": -1}"
+              complete(400, res)
+            }
+          }
+        }
+      }
+    }
+  }
 
+  def main(args: Array[String]): Unit = {
     val logger = LogManager.getLogger(getClass)
     case class CmdParams(serviceHost: String = "0.0.0.0",
                           servicePort: String = "9875",
                           httpsKeyStoreToken: String = "token",
-                          httpsKeyStorePath: String = "./key",
-                          httpsEnabled: Boolean = false
+                          httpsKeyStorePath: String = "./keys/server.p12",
+                          httpsEnabled: Boolean = true,
+                          basePath: String = "./data",
```

Comment:
basePath = /opt/bigdl-as/
when start as as container user can mount existing folder to /opt/bigdl-as/
enrollFilePath = enrolls
policyFilePath = policies
these two files should be encrypted